src/ControlFlow.jl

```diff
@@ -74,9 +74,56 @@ function ReactantCore.traced_if(
     end
 end
 
-function get_region_removing_missing_values(compiled_fn, insertions)
+function ReactantCore.traced_while(cond_fn::CFn, body_fn::BFn, args) where {CFn, BFn}
```

Comment:
Like above, you're not using `CFn` nor `BFn` right?

```suggestion
function ReactantCore.traced_while(cond_fn, body_fn, args)
```